integration-tests/slack/circleci-utils.js

```diff
@@ -13,17 +20,21 @@ const fetchJobArtifacts = async ({ buildNum = process.env.CIRCLE_BUILD_NUM } = {
     throw new Error('**  ERROR: Missing CircleCI build number. Failed to fetch CircleCI job artifacts.');
   }
 
-  // For more arguments and details of the response, see: https://circleci.com/docs/api/v2/index.html#operation/getJobArtifacts
-  const apiUrlRoot = 'https://circleci.com/api/v2/project/github/DataBiosphere/terra-ui';
+  // The v1.1 artifacts API does not need token, so we use it instead.
```

Comment:
Nit: this is already (and more precisely) covered by the comment below: "Because terra-ui is a public repository on GitHub, API token is not required".

```suggestion
```